superbuild/projects_modules/TTK.cmake

```diff
@@ -59,7 +59,7 @@ endif()
 
 set(cmake_args
   ${ep_common_cache_args}
-  -DCMAKE_BUILD_TYPE=${CMAKE_BUILD_TYPE_thirdparts}
+  -DCMAKE_BUILD_TYPE=${CMAKE_BUILD_TYPE_extenals_projects}
```

Comment:
*externals (same for every cmake file here)

Reply:
Done